packages/shopify-codemod/transforms/iife-to-ternary-expression.js

```diff
@@ -0,0 +1,36 @@
+export default function iifeToTernaryExpression({source}, {jscodeshift: j}, {printOptions = {}}) {
+  function isIIFE(path) {
```

Comment:
This name is a bit misleading since an IIFE does not need to be an arrow function, it just happens to be for the particular case we are looking at. Does it matter if the IIFE is passed any parameters?